<launch>

    <node pkg="myahrs_driver" type="myahrs_driver" name="myahrs_driver">
        <param name="port" value="/dev/ttyACM0" />
        <param name="baud_rate" value="115200" />
    </node>

    <node pkg="barc" type="record_experiment.py" name="record_experiment" output="screen">
        <param name="experiment_name" value="traxxas-barc-test" />
        <param name="camera_on" value="false" />
    </node>

</launch>
